action.yml

```diff
@@ -45,6 +45,10 @@ inputs:
     description: 'Interval between execution graph state checks. The value shoud be in seconds.'
     required: false
     default: '30'
+  runtime-parameters-file:
+    description: 'File with the rutine paramers'
+    required: false
```

Comment:
I think this `default` value is wrong here.

Reply:
I forgot to delete that line,  runtime-parameters-file does not need any default value.